command/ca/certificate.go

```diff
@@ -304,9 +296,10 @@ func (f *certificateFlow) GenerateToken(ctx *cli.Context, subject string, sans [
 		}
 	}
 
-	return newTokenFlow(ctx, subject, sans, caURL, root, "", "", "", "", notBefore, notAfter)
+	return newTokenFlow(ctx, subject, sans, caURL, root, "", "", "", "", time.Time{}, time.Time{})
 }
 
+// Sign signs the token using the online or the offline certificate authority.
```

Comment:
Sign signs the CSR using the online or the offline certificate authority. -- right? (CSR not token, or am I confused)